test/conformance/adapters/cuda/cuda_urDeviceCreateWithNativeHandle.cpp

```diff
@@ -7,7 +7,7 @@
 
 using urCudaDeviceCreateWithNativeHandle = uur::urPlatformTest;
 
-TEST_F(urCudaDeviceCreateWithNativeHandle, Success) {
+TEST_F(urCudaDeviceCreateWithNativeHandle, DISABLED_Success) {
```

Comment:
As I understand this test `urDeviceCreateWithNativeHandle` so why we need `cuda_` prefix in the file name? and why we need `cuda` part in the test name, I mean can't we name the test `urDeviceCreateWithNativeHandle` only as testing this entry would mean we already will pass it a cuda handle to test it?

Reply:
You're right about the `cuda_` prefixes in filenames. It's on my list of stuff to do - see [here](https://github.com/oneapi-src/unified-runtime/pull/763#discussion_r1282042701).

The problem with removing `cuda` from the test name is that there already is a test with the name `urDeviceCreateWithNativeHandleTest` as a generic conformance test which it will conflict with.